kalite/contentload/tests/generate_assessment_zips.py

```diff
@@ -0,0 +1,142 @@
+import json
+import os
+import requests
+import shutil
+import tempfile
+import zipfile
+from fle_utils.general import ensure_dir
+from mock import patch, MagicMock
+
+from django.core.management import call_command
+
+from kalite.testing import KALiteTestCase
+from kalite.contentload.management.commands import generate_assessment_zips as mod
+
+TEST_FIXTURES_DIR = os.path.join(os.path.dirname(__file__),
+                                        "fixtures")
+ASSESSMENT_ITEMS_SAMPLE_PATH = os.path.join(TEST_FIXTURES_DIR,
+                                            "assessment_items_sample.json")
+
+
+class GenerateAssessmentItemsCommandTests(KALiteTestCase):
+
+    def setUp(self):
+        self.tempdir_patch = patch.object(tempfile, "gettempdir")
+        self.addCleanup(self.tempdir_patch.stop)
+        self.gettempdir_method = self.tempdir_patch.start()
+
+        # make sure we control the temp dir where temporary images are written
+        self.fake_temp_dir = self.gettempdir_method.return_value = os.path.abspath("tmp/")
+        ensure_dir(self.fake_temp_dir)
+
+    def tearDown(self):
+        shutil.rmtree(self.fake_temp_dir)
+
+    @patch.object(mod, "ASSESSMENT_ITEMS_PATH", ASSESSMENT_ITEMS_SAMPLE_PATH)
+    @patch.object(requests, 'get')
+    def test_command(self, get_method):
+
+        with open(ASSESSMENT_ITEMS_SAMPLE_PATH) as f:
+            assessment_items_content = f.read()
```

Comment:
Beware in case we are dealing with bigger video files. Incidentally, I just merged this in django-dbbackup... it's about reading in chunks: https://bitbucket.org/mjs7231/django-dbbackup/commits/c7e0588c3cc5acd3a96bf7708871edb025b859bc